scheduler/docker/install-standalone.md

```diff
@@ -3,6 +3,12 @@ layout: page
 title: "Deploy Portworx with Docker"
 keywords: portworx, architecture, storage, container, cluster, install, docker, compose, systemd, plugin
 sidebar: home_sidebar
+redirect_from:
+  - /install/docker-plugin.html
+  - /docker/install-px-docker-service.html
+  - /run-with-docker.html
+  - /scheduler/docker-ucp.html
```

Comment:
Is this a better page?
 https://docs.portworx.com/scheduler/docker/install-swarm.html